refresh/templates/common/Package.swift

```diff
@@ -5,8 +5,15 @@ let package = Package(
     targets: [
 <% if (appType === 'crud') { -%>
       Target(name: "<%- applicationModule %>", dependencies: [ .Target(name: "<%- generatedModule %>") ]),
-<% } -%>
-      Target(name: "<%- executableModule %>", dependencies: [ .Target(name: "<%- applicationModule %>") ])
+<% } _%>
+      Target(name: "<%- executableModule %>", dependencies: [ .Target(name: "<%- applicationModule %>") ]),
+<% if (sdkTargets.length > 0) { -%>
+      Target(name: "<%- applicationModule %>", dependencies: [
```

Comment:
Are we going to end up with 2 `Target(name: "Application", ...)` entries if we are generating a CRUD application *and* supplying `sdkTargets`?